src/agent/inspectordebugapi.ts

```diff
@@ -35,7 +35,7 @@ export class BreakpointData {
   constructor(
       public id: string, public active: boolean,
       public apiBreakpoint: apiTypes.Breakpoint,
-      public parsedCondition: estree.Node,
+      public parsedCondition: estree.Node, public locationStr: string,
```

Comment:
nit: The `public locationStr: string` should be on the next line.  If the formatter put it on the line that it is on, we will need to investigate how to configure the formatter.

Reply:
gulp format resolved to same line.